package.json

```diff
@@ -13,6 +13,7 @@
     "build:cleancss": "cleancss -o css/nes.min.css css/nes.css",
     "storybook": "start-storybook -p 6006",
     "build:storybook": "build-storybook",
+    "build-storybook": "npm run storybook",
```

Comment:
Gotcha -- might have been a problem with me working off `master` instead of `develop`, but I was having some problems running that. As long as it works on the main end we're all good.